client/src/pages/login.js

```diff
@@ -1,12 +1,75 @@
+/* eslint-disable react/jsx-no-bind */
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import '../style/login.css';
+import Api from '../services/api';
 
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  validateForm() {
+    const { email } = this.state;
+    const { password } = this.state;
+    return email.length > 0 && password.length > 7;
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: event.target.value,
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    // eslint-disable-next-line no-undef
+    Api.getLogedin(this.state.email, this.state.password);
+    // eslint-disable-next-line react/prop-types
+    this.props.history.push('/');
+  }
+
+
   render() {
     return (
-      <h1>Login Page</h1>
+      <div className="Login">
+        <Form onSubmit={this.handleSubmit.bind(this)}>
+          Login
+          <Form.Group controlId="email">
+            <Form.Control
+              autoFocus
+              name="email"
+              type="email"
+              value={this.state.email}
+              onChange={this.handleChange.bind(this)}
+            />
+          </Form.Group>
+          <Form.Group controlId="password">
+            <Form.Control
+              value={this.state.password}
+              onChange={this.handleChange.bind(this)}
+              type="password"
+            />
+          </Form.Group>
+          <Button
+            block
+            disabled={!this.validateForm()}
+            type="submit"
```

Comment:
![image](https://user-images.githubusercontent.com/6152066/78358838-87bf8b00-75bc-11ea-905a-1700d8f7d51a.png)
You should update seed file according to new restrictions

Reply:
ok its fixed now